Updated joi version and validate function per latest schema.

sheetalkadwal opened the pull request:
**Trello Card:** https://trello.com/c/Wr27WgS2/2-review-and-fix-media-shuttle-metadata-signature-generator

**Commit Summary:**
**16dc506:** Updated joi version and validate function per latest schema.
**6329ca5:** Updated the error handling

sheetalkadwal committed:
Fixed sample code

Updated joi version and validate function per latest schema.

tobsign reviewed (approved):
lgtm. But I'm wondering if checking if error is undefined is really necessary, wrt to `joi` API definition.

sheetalkadwal committed:
Updated the error handling

Updated the error handling

sheetalkadwal commented:
> lgtm. But I'm wondering if checking if error is undefined is really necessary, wrt to `joi` API definition.

Hi @tobsign, Updated the error handling and removed undefined.

tobsign reviewed (approved):
lgtm
